libs/sql-schema-describer/src/mysql.rs

```diff
@@ -77,6 +77,13 @@ impl super::SqlSchemaDescriberBackend for SqlSchemaDescriber<'_> {
         let mut enums = vec![];
         for table_name in &table_names {
             let (table, enms) = self.get_table(table_name, &mut columns, &mut indexes, &mut fks);
+
+            // If we cannot query any of the columns, do not add the table to
+            // the data model...
+            if table.columns.is_empty() {
+                continue;
+            }
```

Comment:
I guess introspecting tables with no columns (as in: does not have any) is not a use case we need to worry about?

Reply:
No.